snippets/base/static/templates/snippetDataWidget.html

```diff
@@ -12,6 +12,7 @@
         {% elif variable.type == types.image %}
           <img src="{{ originalData[variable.name] }}">
           <input type="file" class="image-input">
+          <a href="#" class="image-input-remove">Remove image</a>
```

Comment:
Just a nit, probably doesn't really matter:

If easy to add a 5px left margin/padding to make it line up with the browse button?